src/components/Header/components/NetworkLabel.tsx

```diff
@@ -31,9 +31,13 @@ const useStyles = makeStyles({
   },
 })
 
-const EarlyAccessLabel = () => {
-  const classes = useStyles()
+interface EarlyAccessLabelProps {
+  network?: string
+}
 
+const EarlyAccessLabel = ({ network }: EarlyAccessLabelProps): React.ReactElement => {
+  const classes = useStyles()
+  const formattedNetwork = formatNetwork(network || interfaceNetwork)
```

Comment:
was using interface network as a fallback a requirement?

Reply:
This component is used by  the interface network label and the wallet network as well